### ISSUE TITLE
Refactor build script to use babel-cli

### ISSUE THREAD

**Issue by annekainicUSDS** [usds/us-forms-system#4]
Right now, the way that we transpile the .jsx files into .js is through a build script that runs the files through Babel individually and rewrites them in the `lib/` directory.

The potentially better way to do this is to use the babel-cli package to compile the entire directory at once: https://babeljs.io/docs/usage/cli/#compile-directories. Will probably need to clear the `lib/` directory first, and then re-transpile the directory with each script run.

**Comment by jcmeloni-usds** [comment]
@dmethvin-gov since it seems you already started tracking this down, do you want to own it to completion? if it isn't already that is...

**Comment by dmethvin-gov** [comment]
Yep, it's a pretty easy change. I did it in about a half-hour last week but lost it in an unfortunate smelting accident.
